### ISSUE TITLE
Add Coinpayment transaction Admin Panel Error

### ISSUE THREAD

**Issue by cud4m** [delneg/django-coinpayments#1]
* django-coinpayments version: '0.1.0'
* Django version: 2.0.4
* Python version: 3.6
* Operating System: Debian sid

### Description

I am trying to test everything before forward developing, so I wanted to create a coinpayment transaction, but can´t create any CoinPayments from django admin panel, I think the problem is related to the link_to_payment function in the admin.py.

Not sure anyway, still new to django. And still new to github.  
If someone has this problem, glad to hear. 
I`ll post the fix if I find it.

### What I Did

```'''
Environment:


Request Method: POST
Request URL: http://127.0.0.1:8000/admin/django_coinpayments/coinpaymentstransaction/add/

Django Version: 2.0.4
Python Version: 3.6.5
Installed Applications:
['rest_framework',
 'users.apps.UsersConfig',
 'payments.apps.PaymentsConfig',
 'mrketchain.apps.MarketChainConfig',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'corsheaders',
 'django_coinpayments',
 'product',
 'seller']
Installed Middleware:
['corsheaders.middleware.CorsMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/core/handlers/exception.py" in inner
  35.             response = get_response(request)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  128.                 response = self.process_exception_by_middleware(e, request)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  126.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/contrib/admin/options.py" in wrapper
  574.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  44.         response = view_func(request, *args, **kwargs)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/contrib/admin/sites.py" in inner
  223.             return view(request, *args, **kwargs)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/contrib/admin/options.py" in add_view
  1553.         return self.changeform_view(request, None, form_url, extra_context)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapper
  62.             return bound_func(*args, **kwargs)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/utils/decorators.py" in bound_func
  58.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/contrib/admin/options.py" in changeform_view
  1450.             return self._changeform_view(request, object_id, form_url, extra_context)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/contrib/admin/options.py" in _changeform_view
  1495.                     return self.response_add(request, new_object)

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/contrib/admin/options.py" in response_add
  1098.             current_app=self.admin_site.name,

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/urls/base.py" in reverse
  88.     return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))

File "/home/someuser/venvmarket/lib/python3.6/site-packages/django/urls/resolvers.py" in _reverse_with_prefix
  632.         raise NoReverseMatch(msg)

Exception Type: NoReverseMatch at /admin/django_coinpayments/coinpaymentstransaction/add/
Exception Value: Reverse for 'django_coinpayments_coinpaymentstransaction_change' with arguments '('',)' not found. 1 pattern(s) tried: ['admin/django_coinpayments\\/coinpaymentstransaction\\/(?P<object_id>.+)\\/change\\/$']
```

**Comment by delneg** [comment]
Fixed in the commit mentioned above, released in 0.1.2

**Comment by delneg** [comment]
The problem was unrelated to link_to_payment, but to django's inner resolving of the url for change in django.contrib.admin.options at lines 1095-1099 (Django 2.0.4)
Because of 'id' field being not editable (set to editable=False in the model), it didn't show up in the admin, didn't get populated in the form and the value passed to resolver was ''
I've made id field editable, now you have to fill it in the add form and everything works.
